docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -970,6 +971,24 @@ The values of the `typescript.options.lib` compiler option should be identical t
 *Related configuration file property*: [compilerOptions](configuration-file.md#compileroptions).
 *Related API method*: [runner.compilerOptions](testcafe-api/runner/compileroptions.md)
 
+### --cache
+
+If enabled, the TestCafe proxy caches webpage assets (such as stylesheets and scripts) for the webpages that it processes. The next time the proxy accesses the page, it pulls assets from its cache instead of requesting them from the server.
+
+```sh
+testcafe chrome my-tests --cache
+```
+
+TestCafe emulates the browser's native caching behavior. For example, in *Chrome* TestCafe only caches resources that *Chrome* itself would cache if run without TestCafe.
+
+TestCafe caches scripts, stylesheets, fonts, and other web assets up to **5 MB** in size. TestCafe doesn't cache HTML because that could break [user roles](../guides/advanced-guides/authentication.md#user-roles).
+
+To decrease test run time, enable server-side caching when you test asset-heavy web applications.
```

Comment:
I'm not sure `asset-heavy` is right term because it seems quite informal

Reply:
Admittedly, this is an informal and imprecise term. The problem here is that we don't know how `heavy` the page needs to be to cause problems (it depends on many factors outside of our control). 

So it's on the user to know that their app is `asset-heavy`. 

We can't give a precise number here (or even an approximate one). An alternative is:

```suggestion
If the tested application loads a lot of assets, enable server-side caching to decrease test run time.
```

But, again, `a lot` is not precise nor formal. I think we can allow some informality here.

Reply:
Does this work?

```suggestion
If the tested application loads many heavy assets, enable server-side caching to decrease test run time.
```